### PR TITLE
Updated to run via cron

### PR DESCRIPTION
**Pull request by blarghmatey**
Updated to execute via cron tasks instead of relying on salt to stay connected for the duration of execution

### CODE REVIEW

File: salt/backups/templates/backup_consul.sh [hunk 1]
```diff
@@ -10,3 +10,5 @@ SNAPFILE={{ backupdir }}/consul_snapshot-`date +%Y-%m-%d_%H-%M-%S`.snap
 aws s3 cp $SNAPFILE s3://odl-operations-backups/{{ settings.get('directory', 'consul') }}/
 
 rm $SNAPFILE
+
+salt-call event.fire_master '{"data": "Completed backup of Consul"}' backup/{{ ENVIRONMENT }}/{{ settings.service.title }}/completed
```
Comment (shaidar) [inline salt/backups/templates/backup_consul.sh]: We sure that `settings.service.title` is correct?

**Comment by shaidar** [comment]
@blarghmatey Should this not also include a healthcheck ping to ensure that it's not silently failing?

**Comment by blarghmatey** [comment]
@shaidar the scripts now have healtcheck pings included

**Comment by shaidar** [comment]
👍

**Pull request merged**